stdlib/public/core/EmbeddedRuntime.swift

```diff
@@ -272,11 +272,19 @@ public func swift_dynamicCastClass(object: UnsafeMutableRawPointer, targetMetada
 public func swift_dynamicCastClassUnconditional(object: UnsafeMutableRawPointer, targetMetadata: UnsafeRawPointer,
     file: UnsafePointer<CChar>, line: CUnsignedInt, column: CUnsignedInt) -> UnsafeMutableRawPointer {
   guard let result = swift_dynamicCastClass(object: object, targetMetadata: targetMetadata) else {
-    Builtin.int_trap()
+    fatalError("failed cast")
   }
   return result
 }
 
+@_cdecl("swift_isEscapingClosureAtFileLocation")
+public func swift_isEscapingClosureAtFileLocation(object: Builtin.RawPointer, filename: UnsafePointer<CChar>, filenameLength: Int32, line: Int32, column: Int32, verificationType: CUnsignedInt) -> Bool {
+  guard swift_isUniquelyReferenced_native(object: object) else {
```

Comment:
This will raise a fatal error when `object` is `NULL`, whereas the C++ implementation returns `false` in that case. I'm not sure if it actually matters in practice (is this ever called with `NULL` legitimately?) but it would probably be good to match them.

Reply:
Good point, fixed!